essays/AITBW-roadToDune/index.md

```diff
@@ -0,0 +1,83 @@
+# Dr. Strangelove or: How I handled my stubbornness to become a web developer
+
+Becoming a developer can be a pretty underwhelming task these days, especially if you delve into the [JavaScript](https://hackernoon.com/how-it-feels-to-learn-javascript-in-2016-d3a717dd577f) land, where _the new kid on the block_ can be thrown into obscurity in a matter of days.
```

Comment:
Did you mean `overwhelming`?

Reply:
It might be better to concatenate two or three of these paragraphs into one.

Reply:
Yes, `overwhelming` is the right word. Sorry about that. 🙊

Reply:
Please update the contentan of the markdown 🤗